README.md

```diff
@@ -10,94 +10,94 @@ Pull requests are welcome. Please keep the list in alphabetical order.
+[Elastic](https://www.elastic.co/)                | Elasticsearch as a service               | Java, DevOps, Web                              | ✓                                    | ?                       | [Jobs](https://www.elastic.co/about/careers)<br>[Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Elastic-EI_IE751551.11,18.htm)
+[Envato](https://envato.com/)                     | Sites for creatives                      | Ruby, Web                                      | ✓                                    | ?                       | [Jobs](https://envato.com/careers/)<br>[Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Envato-EI_IE700841.11,17.htm)
+[Eyeo](https://eyeo.com)                          | AdBlock Plus & More                      | Many                                           | ?                                    | ?                       | [Jobs](https://eyeo.com/jobs/)<br>[Glassdoor](https://www.glassdoor.com/Overview/Working-at-Eyeo-EI_IE848298.11,15.htm)
+[Fairwinds](https://www.fairwinds.com/)           | Kubernetes Services and Software         | Go, Kubernetes, Terraform                      | ✓                                    | ?                       | [Jobs](https://www.fairwinds.com/careers)
+[Fire Engine Red](http://fire-engine-red.com/)    | Education market solutions               | ?                                              | ?                                    | ?                       | [Jobs](http://fire-engine-red.com/about/#careers)<br>[Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Fire-Engine-RED-EI_IE1339735.11,26.htm)
+[Four Kitchens](http://www.fourkitchens.com/)     | Web consulting                           | Drupal, Node, PHP, Web                         | ?                                    | ?                       | [Jobs](http://www.fourkitchens.com/careers/)<br>[Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Four-Kitchens-EI_IE588769.11,24.htm)
+[Genuitec](http://www.genuitec.com)               | Eclipse-based tools                      | Java, Eclipse, Web, Mobile                     | ?                                    | ?                       | [Jobs](http://www.genuitec.com/company/careers/)<br>[Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Genuitec-EI_IE346957.11,19.htm)
+[Ghost](https://ghost.org/)                       | Publishing platform                      | Node, Web                                      | ✓                                    | Anywhere                | [Jobs](https://ghost.org/about/)
+[GitHub](https://github.com)                      | GitHub                                   | Git, Ruby, Web                                 | ✓                                    | Mostly Anywhere         | [Jobs](https://github.com/about/jobs)<br>[Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-GitHub-EI_IE671945.11,17.htm)
```

Comment:
What does "mostly anywhere" mean? :slightly_smiling_face:

Reply:
If you check their website, they don't explicitly say they hire from anywhere but on the other hand they list `Americas`, `Europe`, `Asia Pacific`... Which cover a good chunk of the planet... :-)

Reply:
I think we can go with `Anywhere` then. :slightly_smiling_face: